test/functional/apps/visualize/replaced_vislib_chart_types/_line_chart_split_chart.ts

```diff
@@ -203,18 +203,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.visEditor.changeYAxisFilterLabelsCheckbox(axisId, false);
         await PageObjects.visEditor.clickGo(true);
         const labels = await PageObjects.visChart.getYAxisLabels(xyChartSelector);
-        const expectedLabels = [
-          '0',
-          '1,000',
-          '2,000',
-          '3,000',
-          '4,000',
-          '5,000',
-          '6,000',
-          '7,000',
-          '8,000',
-          '9,000',
-        ];
+        const expectedLabels = ['0', '2,000', '4,000', '6,000', '8,000'];
```

Comment:
@nickofthyme why do we get less labels here?

Reply:
We reduced the default number of ticks in the Y axis https://github.com/elastic/elastic-charts/pull/2005
10 was too much in most situation so we reduced it to 5 that usually is a good compromise until we don't have a dynamic system in place